OKTA-794271 Acrolinx edits for Work with Okta Session cookies (#5049)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/session-cookie/main/index.md

title: Work with Okta session cookies
---

> > **Note:** This document is written for Okta Classic Engine. If you're using Okta Identity Engine, see [Understand how sessions work after the upgrade](/docs/guides/oie-upgrade-sessions-api/) and this [Sessions APIs](/docs/guides/ie-limitations/main/#sessions-apis) section. See [Identify your Okta solution](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) to determine your Okta version.

Reviewer: ```suggestion
> **Note:** This page provides information on Okta Classic Engine. If you're using Okta Identity Engine, see [Understand how sessions work after the upgrade](/docs/guides/oie-upgrade-sessions-api/) and this [Sessions APIs](/docs/guides/ie-limitations/main/#sessions-apis) section. See [Identify your Okta solution](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) to determine your Okta version.
```
